app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -659,6 +660,12 @@ def is_mfs_same_home
         is_same_home = @intake.tenant_same_home_spouse_yes? || @intake.homeowner_same_home_spouse_yes?
         is_mfs && is_same_home
       end
+
+      def applicable_1099rs
```

Comment:
[dust] Does applicable mean applicable to the state taxes in general? Or just to line 20? If it's just line 20 I wonder if this should be moved into nj 1040 calc helpers

Reply:
@mmazanec22 I'm not sure about the broadness question, but a good point about the term "applicable" being vague, I think we could rename it to "non-military 1099-Rs" to be clearer!